static/simple-portfolio-template/index.html

```diff
@@ -0,0 +1,344 @@
+
+        <p class="mt-10 text-xl leading-7">
+          It is a long established fact that a reader will be distracted by the
+          readable content of a page when looking at its layout. The point of
+          using LoremIt is a long established fact that a reader will be
+          distracted by the readable content of a page when looking at its
+          layout. The point of using Lorem
+        </p>
+      </article>
+    </section>
+
+    <section class="flex flex-col items-center py-2 pb-4" id="blogs-section">
+      <span class="h-40 w-px bg-slate-800"></span>
+      <h3 class="section-title">Checkout My Blog</h3>
+
+      <article class="my-20 flex w-5/6 flex-col gap-8 md:flex-row lg:w-3/4">
+        <div class="blog-container">
+          <img src="./assets/images/b-1.jpg" alt="Blog 1" />
+          <h4 class="blog-title">Readable content of a page</h4>
+          <p class="blog-desc">
+            It is a long established fact that a reader will be distracted by
+            the readable content of a page when looking at its layout. The point
+            of using Lorem
+          </p>
+        </div>
+
+        <div class="blog-container">
+          <img src="./assets/images/b-2.jpg" alt="Blog 2" />
+          <h4 class="blog-title">Readable content of a page</h4>
+          <p class="blog-desc">
+            It is a long established fact that a reader will be distracted by
+            the readable content of a page when looking at its layout. The point
+            of using Lorem
+          </p>
+        </div>
+
+        <div class="blog-container">
+          <img src="./assets/images/b-3.jpg" alt="Blog 3" />
+          <h4 class="blog-title">Readable content of a page</h4>
+          <p class="blog-desc">
+            It is a long established fact that a reader will be distracted by
+            the readable content of a page when looking at its layout. The point
+            of using Lorem
+          </p>
+        </div>
+      </article>
+    </section>
+
+    <section
+      class="flex flex-col items-center bg-neutral-50 py-14"
+      id="contact-section"
+    >
+      <h3 class="section-title">Contact Me</h3>
+
+      <form class="mt-10 flex w-5/6 flex-col justify-center md:w-3/4">
+        <input type="text" placeholder="Full Name" class="input" />
+        <input type="tel" placeholder="Phone Number" class="input" />
+        <input type="email" placeholder="E-Mail Address" class="input" />
+        <textarea rows="10" placeholder="Comments" class="input"></textarea>
+        <button
+          class="mx-auto mt-5 rounded bg-yellow-400 py-3 px-10 uppercase tracking-wide text-white shadow active:shadow-none"
+        >
+          Send
+        </button>
+      </form>
+    </section>
+
+    <footer
+      class="flex flex-col items-center justify-center bg-slate-900 py-14"
+    >
+      <h2 class="text-4xl font-semibold uppercase text-white">Let's Connect</h2>
+
+      <article class="mt-8 flex gap-x-10">
```

Comment:
Those are navigation links, so it should probably be wrapped in nav, & links itself are a list, so ul+ li, like you did in the navbar above